### ISSUE TITLE
Yellow color doesn't have enough contrast against white background

### ISSUE THREAD

**Issue by jagtalon** [duckduckgo/duckduckgo-privacy-extension#273]
Yellow doesn't have enough contrast on the white background. Also hard to read.

<img width="1680" alt="" src="https://user-images.githubusercontent.com/81969/43593115-08cc8030-9645-11e8-8dd4-3c66b3e456a1.png">
